packages/calypso-products/src/plans-list.tsx

```diff
@@ -1764,6 +1792,17 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getStoreSlug: () => PLAN_JETPACK_COMPLETE,
 		getPathSlug: () => 'complete',
 		getProductId: () => 2014,
+		getWhatIsIncluded: () => [
+			translate( 'Real-time backups as you edit' ),
+			translate( '1TB (1,000GB) of cloud storage' ),
+			translate( '1-year activity log archive' ),
+			translate( 'Unlimited one-click restores from the last 1 year' ),
+			translate( 'Real-time malware scanning and one-click fixes ' ),
```

Comment:
Translation strings should not start or end with spaces.

```suggestion
			translate( 'Real-time malware scanning and one-click fixes' ),
```